### ISSUE TITLE
openvpn client + danted

### ISSUE THREAD

**Issue by vinnitu** [vimagick/dockerfiles#6]
Can You help me merge openvpn client with sockd server?

Ideally I need pass .ovpn conf with credentials and config to container and setup local port for socks server

**Comment by vimagick** [comment]
You can compose these two containers.
```
OpenVPN Client -> Dante Server -> OpenVPN Server
```
Set [`socks-proxy` or `http-proxy`][1] in your `client.ovpn`.

[1]: https://community.openvpn.net/openvpn/wiki/Openvpn23ManPage

**Comment by vinnitu** [comment]
no, I mean openvpn-server will socks5-server(sockd) simultaneously, not client

**Comment by vimagick** [comment]
You can run openvpn and dante side by side.
